2014/01/02 15:36:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/setaflow/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/setaflow/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /var/www/setaflow/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /var/www/setaflow/framework/base/CModule.php(387): CDbConnection->init()
#3 /var/www/setaflow/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/setaflow/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/setaflow/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/setaflow/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/setaflow/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/setaflow/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/setaflow/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/setaflow/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/setaflow/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/giixModel/i...')
#14 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#15 /var/www/setaflow/app/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/setaflow/app/index.php/gii/giixModel/index
HTTP_REFERER=http://localhost/setaflow/app/index.php/gii/default/index
---
2014/01/02 15:58:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "forms" was not found.' in /var/www/setaflow/framework/web/actions/CViewAction.php:112
Stack trace:
#0 /var/www/setaflow/framework/web/actions/CViewAction.php(123): CViewAction->resolveView('forms')
#1 /var/www/setaflow/framework/web/actions/CAction.php(76): CViewAction->run()
#2 /var/www/setaflow/framework/web/CController.php(308): CAction->runWithParams(Array)
#3 /var/www/setaflow/framework/web/CController.php(286): CController->runAction(Object(CViewAction))
#4 /var/www/setaflow/framework/web/CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('page')
#6 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/page')
#7 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#8 /var/www/setaflow/app/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/setaflow/app/index.php/site/page?view=forms
HTTP_REFERER=http://localhost/setaflow/app/index.php/site/page?view=tables
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/loa...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/clo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:29:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/site/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/site/page?view=columns
---
2014/01/02 16:29:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/site/page?view=columns
---
2014/01/02 16:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/site/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/site/page?view=elements
---
2014/01/02 16:31:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/site/page?view=elements
---
2014/01/02 16:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small5.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small5...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small4.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small4...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small2.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small2...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small3.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small3...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small1.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small1...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small2.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small2...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small4.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small4...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small3.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small3...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small1.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small1...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/blog/small5.jpg".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/blog/small5...')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/loading.gif".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/loading.gif')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "img/close.png".' in /var/www/setaflow/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('img/close.png')
#1 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/setaflow/app/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/setaflow/app/index.php/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/clo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/loa...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/clo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/loa...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small5.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small1.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small4.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small3.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/blo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/blog/small2.jpg
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/clo...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/close.png
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:32:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "img".' in /var/www/setaflow/framework/web/CController.php:483
Stack trace:
#0 /var/www/setaflow/framework/web/CController.php(270): CController->missingAction('img')
#1 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('img')
#2 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('tblUser/img/loa...')
#3 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/setaflow/app/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/setaflow/app/index.php/tblUser/img/loading.gif
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/01/02 16:33:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "elements" was not found.' in /var/www/setaflow/framework/web/actions/CViewAction.php:112
Stack trace:
#0 /var/www/setaflow/framework/web/actions/CViewAction.php(123): CViewAction->resolveView('elements')
#1 /var/www/setaflow/framework/web/actions/CAction.php(76): CViewAction->run()
#2 /var/www/setaflow/framework/web/CController.php(308): CAction->runWithParams(Array)
#3 /var/www/setaflow/framework/web/CController.php(286): CController->runAction(Object(CViewAction))
#4 /var/www/setaflow/framework/web/CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/setaflow/framework/web/CWebApplication.php(282): CController->run('page')
#6 /var/www/setaflow/framework/web/CWebApplication.php(141): CWebApplication->runController('site/page')
#7 /var/www/setaflow/framework/base/CApplication.php(180): CWebApplication->processRequest()
#8 /var/www/setaflow/app/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/setaflow/app/index.php/site/page?view=elements
HTTP_REFERER=http://localhost/setaflow/app/index.php/tblUser/admin
---
2014/02/10 05:38:08 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "D:\xampp\htdocs\hrdc\hr\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in D:\xampp\htdocs\hrdc\framework\web\CAssetManager.php:138
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('D:\xampp\htdocs...')
#1 D:\xampp\htdocs\hrdc\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 D:\xampp\htdocs\hrdc\framework\zii\widgets\grid\CGridView.php(350): CAssetManager->publish('D:\xampp\htdocs...')
#3 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(147): CGridView->init()
#4 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.gri...', Array)
#5 D:\xampp\htdocs\hrdc\hr\themes\abound\views\site\index.php(132): CBaseController->widget('zii.widgets.gri...', Array)
#6 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#7 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', NULL, true)
#8 D:\xampp\htdocs\hrdc\framework\web\CController.php(869): CBaseController->renderFile('D:\xampp\htdocs...', NULL, true)
#9 D:\xampp\htdocs\hrdc\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#10 D:\xampp\htdocs\hrdc\hr\protected\controllers\SiteController.php(32): CController->render('index')
#11 D:\xampp\htdocs\hrdc\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#12 D:\xampp\htdocs\hrdc\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 D:\xampp\htdocs\hrdc\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 D:\xampp\htdocs\hrdc\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('')
#16 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#17 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/hrdc/hr/
---
2014/02/10 07:24:49 [error] [exception.CException] exception 'CException' with message 'Property "CMenu.showScriptName" is not defined.' in D:\xampp\htdocs\hrdc\framework\base\CComponent.php:173
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWidgetFactory.php(161): CComponent->__set('showScriptName', false)
#1 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'zii.widgets.CMe...', Array)
#2 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CMe...', Array)
#3 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\tpl_navigation.php(39): CBaseController->widget('zii.widgets.CMe...', Array)
#4 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\main.php(56): require_once('D:\xampp\htdocs...')
#5 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#6 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\htdocs\hrdc\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\htdocs\hrdc\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#9 D:\xampp\htdocs\hrdc\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#10 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(206): COutputProcessor->run()
#11 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#12 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\column1.php(14): CBaseController->endContent()
#13 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#14 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#15 D:\xampp\htdocs\hrdc\framework\web\CController.php(784): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#16 D:\xampp\htdocs\hrdc\hr\protected\controllers\SiteController.php(32): CController->render('index')
#17 D:\xampp\htdocs\hrdc\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#18 D:\xampp\htdocs\hrdc\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 D:\xampp\htdocs\hrdc\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#20 D:\xampp\htdocs\hrdc\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('')
#22 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#23 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#24 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#25 {main}
REQUEST_URI=/hrdc/hr/
HTTP_REFERER=http://localhost/hrdc/hr/
---
2014/02/10 07:26:05 [error] [exception.CException] exception 'CException' with message 'Property "CMenu.showScriptName" is not defined.' in D:\xampp\htdocs\hrdc\framework\base\CComponent.php:173
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWidgetFactory.php(161): CComponent->__set('showScriptName', false)
#1 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(SiteController), 'zii.widgets.CMe...', Array)
#2 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(172): CBaseController->createWidget('zii.widgets.CMe...', Array)
#3 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\tpl_navigation.php(39): CBaseController->widget('zii.widgets.CMe...', Array)
#4 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\main.php(56): require_once('D:\xampp\htdocs...')
#5 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#6 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#7 D:\xampp\htdocs\hrdc\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#8 D:\xampp\htdocs\hrdc\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#9 D:\xampp\htdocs\hrdc\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#10 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(206): COutputProcessor->run()
#11 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#12 D:\xampp\htdocs\hrdc\hr\themes\hr\views\layouts\column1.php(14): CBaseController->endContent()
#13 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(126): require('D:\xampp\htdocs...')
#14 D:\xampp\htdocs\hrdc\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\xampp\htdocs...', Array, true)
#15 D:\xampp\htdocs\hrdc\framework\web\CController.php(784): CBaseController->renderFile('D:\xampp\htdocs...', Array, true)
#16 D:\xampp\htdocs\hrdc\hr\protected\controllers\SiteController.php(32): CController->render('index')
#17 D:\xampp\htdocs\hrdc\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#18 D:\xampp\htdocs\hrdc\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#19 D:\xampp\htdocs\hrdc\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#20 D:\xampp\htdocs\hrdc\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#21 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('')
#22 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#23 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#24 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#25 {main}
REQUEST_URI=/hrdc/hr/
HTTP_REFERER=http://localhost/hrdc/hr/
---
2014/02/12 02:40:38 [error] [exception.CException] exception 'CException' with message 'TestController cannot find the requested view "index".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:878
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(782): CController->renderPartial('index', NULL, true)
#1 D:\xampp\htdocs\hrdc\hr\protected\controllers\TestController.php(32): CController->render('index')
#2 D:\xampp\htdocs\hrdc\framework\web\actions\CInlineAction.php(49): TestController->actionIndex()
#3 D:\xampp\htdocs\hrdc\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\hrdc\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\hrdc\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('')
#7 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('test')
#8 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/hrdc/hr/test
---
2014/02/12 03:45:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "division".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('division')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/division
---
2014/02/12 03:45:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "division".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('division')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/division
---
2014/02/12 03:46:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Division".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('Division')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/Division
---
2014/02/12 03:46:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "division".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('division')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/index.php/division
---
2014/02/20 04:56:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gix".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('gix')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/gix
---
2014/02/20 05:02:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "xgii".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('xgii')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/xgii
---
2014/02/20 05:22:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/user
---
2014/02/20 05:23:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "project".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('project')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/project
---
2014/02/20 09:24:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "game".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/
---
2014/02/20 09:43:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "humanresource".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('humanresource')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/humanresource
---
2014/02/20 09:43:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/humanResource
---
2014/02/20 09:43:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "human_resource".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('human_resource')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/human_resource
---
2014/02/20 09:43:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "human_resources".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('human_resources')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/human_resources
---
2014/02/20 09:44:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hurmanresources".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('hurmanresources')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/hurmanresources
HTTP_REFERER=http://localhost/hrdc/hr/humanresources
---
2014/02/20 09:44:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hurmanresources".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('hurmanresources')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/hurmanresources
HTTP_REFERER=http://localhost/hrdc/hr/humanresources
---
2014/02/20 09:54:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "projecs".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projecs')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/projecs
---
2014/02/20 10:06:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "themes/hr/js/quick_search.js".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('themes/hr/js/qu...')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/themes/hr/js/quick_search.js
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 10:06:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "themes/hr/js/quick_search.js".' in D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('themes/hr/js/qu...')
#1 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/hrdc/hr/themes/hr/js/quick_search.js
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:00:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=r&limit=10&offset=0&_=1392890431966
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:00:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rr&limit=10&offset=0&_=1392890437611
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:00:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rrt&limit=10&offset=0&_=1392890438398
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=r&limit=10&offset=0&_=1392890491271
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rt&limit=10&offset=0&_=1392890491421
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=r&limit=10&offset=0&_=1392890495502
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rt&limit=10&offset=0&_=1392890495762
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rtt&limit=10&offset=0&_=1392890496150
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rttt&limit=10&offset=0&_=1392890496640
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rtt&limit=10&offset=0&_=1392890497699
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=rt&limit=10&offset=0&_=1392890497946
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:01:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=r&limit=10&offset=0&_=1392890498466
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
2014/02/20 11:20:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "humanResource".' in D:\xampp\htdocs\hrdc\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\hrdc\framework\web\CController.php(270): CController->missingAction('humanResource')
#1 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(282): CController->run('humanResource')
#2 D:\xampp\htdocs\hrdc\framework\web\CWebApplication.php(141): CWebApplication->runController('projects/humanR...')
#3 D:\xampp\htdocs\hrdc\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\xampp\htdocs\hrdc\hr\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/hrdc/hr/projects/humanResource/quickSearch?q=kdjf&limit=10&offset=0&_=1392891647953
HTTP_REFERER=http://localhost/hrdc/hr/projects/admin
---
